app/models/sub_service_request.rb

```diff
@@ -315,19 +314,15 @@ def is_complete?
     return FINISHED_STATUSES.include?(status)
   end
 
```

Comment:
Removed parameter on `find_editable_id` which allowed us to use eager loading to load organizations prior to calling this method